clean/grand_isle_pd.py

```diff
@@ -1,3 +1,142 @@
+<<<<<<< HEAD
```

Comment:
Looks like unresolved merge conflict to me.

Reply:
No conflict errors are appearing for me.

Reply:
@ayyubibrahimi `<<<<<<< HEAD` this is merge conflict indicator. Do you need some help?